library/Exceptions/NestedValidationExceptionInterface.php

```diff
@@ -0,0 +1,8 @@
+<?php
+namespace Respect\Validation\Exceptions;
+
+interface NestedValidationExceptionInterface extends ValidationExceptionInterface
```

Comment:
Since `ValidationExceptionInterface` will extend `ExceptionInterface` after I make some changes, do you still want me to also extend `ExceptionInterface` here?